fetch/handler.go

```diff
@@ -46,24 +46,20 @@ func newHandler(
 
 // handleMaliciousIDsReq returns the IDs of all known malicious nodes.
 func (h *handler) handleMaliciousIDsReq(ctx context.Context, _ []byte) ([]byte, error) {
-	nodes, err := identities.GetMalicious(h.cdb)
+	nodes, err := identities.All(h.cdb)
```

Comment:
It sounds like _all_ existing identities are picked, not only malicious. Perhaps the pkg should be named `malicious` instead?

Reply:
I'm not sure about renaming the `identities` package/db table, since I plan on getting rid of it soon anyway and the replacement for it is already called `malfeasance`. I renamed `All` to `AllMalicious` though to make clearer that only malicious identities are returned from the function 🙂